backend/src/main/java/de/bund/digitalservice/ris/norms/application/service/BucketService.java

```diff
@@ -0,0 +1,87 @@
+package de.bund.digitalservice.ris.norms.application.service;
```

Comment:
Shouldn't this be an output adapter? So similar to the database? I don't think we should care in the application layer about what kind of storage solution we use for the published norms and just call a port.

Reply:
Jep you are so right! I added two output ports (private and public data)